First draft of article (#1)
arfc/2021-bachmann-epjn: introduction.tex

\section{Introduction}

> Most of the \glspl{LWR} operating the U.S. are slated to retire

Reviewer: ```suggestion
Most of the \glspl{LWR} operating in the U.S. are slated to retire
```

before 2050; meaning that if nuclear power is to continue providing a 
significant portion of energy in the U.S., new reactors will need to be built. 
New reactors are likely to be an advanced reactor design, many of 
which require \gls{HALEU} to fuel them. \gls{HALEU} is uranium that 
is enriched between 5-20\% $^{235}$U, compared to the less than 5\% $^{235}$U 
that fuels current \glspl{LWR}. \gls{HALEU} fuel will help 
the reactors achieve higher burnups and longer cycle times than current 
\glspl{LWR}. However, changes to the fuel enrichment are 
expected 
to change the material requirements of the fuel cycle, and may delay 
new reactor design deployment.

The current supply chain for \gls{LEU} below 5\% enrichment is based on 
enriching \gls{NU} to the required level, and there is no commercial 
supply chain for \gls{HALEU}. There are
two primary methods for creating \gls{HALEU}: \gls{HEU} recovery and 
down-blending, 
and the enrichment of \gls{NU}. Downblending \gls{HEU} is 
limited by the physical inventory of \gls{HEU} available. 
Enriching \gls{NU} is limited by the amount of uranium that can be 

> mined and \gls{SWU} capacity to enrich the urnaium. Understanding the 

Reviewer: ```suggestion
mined and the \gls{SWU} capacity required to enrich the uranium. Understanding the 
```

requires about 20,000 MT of \gls{NU} and about 600 MT of fuel each year 
to produce 100 GWe-yr of electricity at equilibrium. This is more than the 
almost 19,000 MT of \gls{NU} and about 2,000 MT of uranium fuel at 4.21\%
enrichment that would be required in \gls{EG}01 - ``Once-through using enriched-U 
fuel in thermal critical reactors'', which models the continued use of current 
\glspl{LWR}. This shows an increase in \gls{NU} and fuel mass is needed to 
deploy reactors using \gls{HALEU}.
A similar effect was observed by increasing the enrichment of fuel in a 
\gls{PWR} from 4\% to 7\% \cite{burns_reactor_2020}, and in a \gls{LWR}
\gls{SMR} of Russian origin \cite{hernandez_potential_2020}. 

Based on this prior work, increasing the enrichment level of uranium fuel 
is expected to  
increase the material requirements at the front end of 
the fuel cycle. However, the exact change in requirements depends 
on the type of reactor, enrichment level, and transition scenario.
Therefore, it is important to accurately model the transition to any future
reactor designs to quantify the material requirements and understand

> if current supply chain is sufficient to meet them. 

Reviewer: ```suggestion
if the current supply chain is sufficient to meet them. 
```
